tuftool/fips-test/README.md

```diff
@@ -0,0 +1,115 @@
+# Testing tough FIPS feature
+
+The following are steps to take to test FIPS support with `tough` and `tuftool`. Steps will walk
+through creating a TUF repo, hosting the repo via Docker containers running nginx, and testing 
+`tuftool download` against servers using FIPS and non-FIPS ciphers. Creation of the TUF repo
+relies on AWS KMS for signing.
+
+## Install tuftool
+
+```sh
+# From latest release
+$ cargo install --force tuftool --all-features
+```
+
+```sh
+# From local changes
+$ cargo install --path tuftool --all-features
```

Comment:
nit: the rest of the commands in this runbook assume that you are currently in the `tuftool/fips-test` repo, so it could be good to match that here:

```suggestion
$ cargo install --path ../../tuftool --all-features
```

(could also do the below, but the command is less readable...)

```suggestion
$ cargo install --path .. --all-features
```